logback-classic/src/main/java/ch/qos/logback/classic/pattern/SyslogStartConverter.java

```diff
@@ -29,21 +30,36 @@
   long lastTimestamp = -1;
   String timesmapStr = null;
   SimpleDateFormat simpleFormat;
-  String localHostName;
+  String localHostName = null;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2371638216/issues/source?bid=3222618&fileBranchId=3222706#l33)